<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testsuite errors="0" failures="1" name="io.cucumber.core.plugin.JUnitFormatter" skipped="0" tests="1" time="3.915">
    <testcase classname="US1013 Kullanici DB Util ile concort Hotel database'ini test eder" name="TC19 DB Util ile Concort Hotel Database Read Test" time="3.188">
        <failure message="" type="java.lang.AssertionError">
            <![CDATA[Given kullanici DBUtill ile CHQA database'ine baglanir......................passed
And kullanici DBUtill ile "tHOTELROOM" tablosundaki "Price" verilerini alir.passed
And kullanici DBUtill ile "Price" sutunundaki 5. fiyatin 400 oldugunu test eder.failed

StackTrace:
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at dbStepdefinitions.DbUtilStepDefinitions.kullanici_db_utill_ile_sutunundaki_fiyatin_oldugunu_test_eder(DbUtilStepDefinitions.java:42)
	at âœ½.kullanici DBUtill ile "Price" sutunundaki 5. fiyatin 400 oldugunu test eder(file:///Users/muhammetyuksel/Desktop/com.batch30Cucumber/src/test/resources/features/US1013_DbUtilTests.feature:7)
]]>
        </failure>
    </testcase>
</testsuite>
